frontend/src/tbpro/elements/TextInput.vue

```diff
@@ -48,7 +48,7 @@ const props = withDefaults(defineProps<Props>(), {
   maxLength: null,
 });
 
-defineEmits(['submit']);
+const emit = defineEmits(['submit', 'blur']);
 defineExpose({ focus });
```

Comment:
I think you can expose blur
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/blur

then you can just grab the ref and call .blur. But the event also works.

Reply:
(I think the emit is fine)